site/scripts/add-code-snippets-oas.js

```diff
@@ -58,12 +59,19 @@ const EXTENSION_TO_LANG = {
 
 var spec_name = '';
 var operation_id = '';
-const SPEC_SOURCE_DIRECTORY = 'specs-source/';
+const SPEC_SOURCE_DIRECTORY = 'specs/';
```

Comment:
This could be cleaned up a bit by using a single variable since the source and destination are now the same.